src/workerd/io/worker-interface.capnp

```diff
@@ -290,12 +290,13 @@ struct Trace @0x8e8d911203762d34 {
       hibernate @5 :Hibernate;
       spanOpen @6 :SpanOpen;
       spanClose @7 :SpanClose;
-      attribute @8 :List(Attribute);
-      return @9 :Return;
-      diagnosticChannelEvent @10 :DiagnosticChannelEvent;
-      exception @11 :Exception;
-      log @12 :Log;
-      link @13 :Link;
+      completedSpan @8 :UserSpanData;
```

Comment:
Because this is planned to be deleted, move this to the end of the list and add a big comment saying this will be deleted please.
Also explain why we are adding it now and when we can delete it.

Reply:
👍 updated the capnp definition